### PR TITLE
Fix bad surface extend added in previous pull request!

### PR DESCRIPTION
**Pull request by 2020karaulov2020**
@wootguy  Hotfix ! ))  (Previous pull request broken all maps)

### CODE REVIEW

File: src/qtools/rad.cpp [hunk 1]
```diff
@@ -358,8 +357,7 @@ void CalcFaceExtents(Bsp* bsp, lightinfo_t* l)
 
 	s = l->face;
 
-	mins[0] = mins[1] = FLT_MIN;
-	maxs[0] = maxs[1] = FLT_MAX;
+	bool initminmax = false;
```
Comment (wootguy) [inline src/qtools/rad.cpp:361]: Only change in this commit should be reverting this change. It used to be:
```
mins[0] = mins[1] = FLT_MAX;
maxs[0] = maxs[1] = FLT_MIN;
```
That should fix the bad surface extents (min/max were swapped).

**Review by wootguy — CHANGES_REQUESTED** [review]

File: src/qtools/rad.cpp (hunk 1 above)
Comment (2020karaulov2020) [reply src/qtools/rad.cpp:361]: No, You can test, FLT_MAX/FLT_MIN broke all maps ))

Comment (2020karaulov2020) [reply src/qtools/rad.cpp:361]: (Cause random lightmaps and if select any face, it write 'surface extents')

Comment (wootguy) [reply src/qtools/rad.cpp:361]: ok I tried without FLT_MIN/MAX and still broken. Maybe revert all changes in this file? The initminmax stuff might be less efficient, and this code needs to be as fast as possible.

Comment (2020karaulov2020) [reply src/qtools/rad.cpp:361]: @wootguy previous values :

**mins[0] = mins[1] = 999999;
maxs[0] = maxs[1] = -99999;**

and 

**mins[0] = mins[1] = 99999999;
maxs[0] = maxs[1] = -99999999;**

It works. But if replace it to FLT_MAX FLT_MIN, it cause problems in all maps.

**If just initialize mins/maxs with first value from list, it works in all maps.**

Comment (2020karaulov2020) [reply src/qtools/rad.cpp:361]: I checked this method works fine without FLT_MIN / FLT_MAX;

Comment (wootguy) [reply src/qtools/rad.cpp:361]: wow that is strange. I just noticed there were multiple places were FLT_MIN/MAX were used here. Reverting both of those changes fixed it.

Comment (wootguy) [reply src/qtools/rad.cpp:361]: Ok, can you update the request to only revert these lines then?
https://github.com/wootguy/bspguy/blob/master/src/qtools/rad.cpp#L305
https://github.com/wootguy/bspguy/blob/master/src/qtools/rad.cpp#L362

Comment (2020karaulov2020) [reply src/qtools/rad.cpp:370]: I can just increase code speed ? https://github.com/wootguy/bspguy/pull/34/commits/d92d5155f06e8b18d211961a53ddf9d3049f237f

Comment (2020karaulov2020) [reply src/qtools/rad.cpp:370]: I just don't understand why there is an error with FLT_MIN / MAX, but this totally fixes it))

Comment (2020karaulov2020) [reply src/qtools/rad.cpp:370]: How I can test map load speed with these and old code ?

File: src/qtools/rad.cpp [hunk 2]
```diff
@@ -381,15 +388,24 @@ void CalcFaceExtents(Bsp* bsp, lightinfo_t* l)
...
+				{
+					mins[j] = val;
+				}
+				if (val > maxs[j])
+				{
+					maxs[j] = val;
+				}
 			}
 		}
+		initminmax = true;
```
Comment (wootguy) [inline src/qtools/rad.cpp:408]: would rather this `initminmax` stuff be removed entirely. The rad code is magical and I don't to risk breaking anything.

**Review by wootguy — CHANGES_REQUESTED** [review]

File: src/qtools/rad.cpp (hunk 1 above)
Comment (wootguy) [reply src/qtools/rad.cpp:370]: merging maps is a good way to benchmark, which does a lot of lightmap calculations.

File: src/qtools/rad.cpp (hunk 2 above)
Comment (2020karaulov2020) [reply src/qtools/rad.cpp]: Ok now changes reverted back!

Comment (wootguy) [reply src/qtools/rad.cpp]: Thanks! If this project ever gets unit tests then changes like this will be less risky.

**Pull request merged**